policy/utils/openshift.rego

```diff
@@ -1,5 +1,7 @@
 package utils.openshift
 
+import data.utils.kubernetes
```

Comment:
this was what I was talking about in my comment

Reply:
because its doing an OR:
- https://github.com/redhat-cop/rego-policies/pull/43/files#diff-0d550edf0aa9801f8d7d11a89af02088R19-R25

it was either add k8s dep into openshift, or create another util which was for both. went with the simpler route